### ISSUE TITLE
[Suggestion]: Applied Energistics Growth Accelerators

### ISSUE THREAD

**Issue by Daveawb** [Mrbysco/GeOre#5]
### General Info

- [ ] I would like to propose new content
- [ ] I would like to propose changes
- [X] I would like to propose another thing

### Suggestion Description

Minecraft 1.21.1
NeoForge 21.1.69
GeOre 6.0.4

Growth accelerators don't work for GeOres but work well for Applied Energistics own budding blocks and Amethyst Budding blocks. It would stand to reason that GeOre should probably support them (if possible).

**Comment by ShyNieke** [comment]
is this issue confirmed as fixed if so can we close this issue?

**Comment by DavidBarker-EE** [comment]
Yes, it's confirmed as fixed. I have not tested it but I can see that the missing tags are indeed available now for growth accelerators in AE.
